primefaces/src/main/frontend/packages/components/src/tooltip/tooltip.widget.js

```diff
@@ -64,8 +64,8 @@ PrimeFaces.widget.Tooltip = class Tooltip extends PrimeFaces.widget.BaseWidget {
     init(cfg) {
         super.init(cfg);
 
-        this.cfg.showEvent = this.cfg.showEvent ? this.cfg.showEvent + '.tooltip' : 'mouseenter.tooltip';
-        this.cfg.hideEvent = this.cfg.hideEvent ? this.cfg.hideEvent + '.tooltip' : 'mouseleave.tooltip';
+        this.cfg.showEvent = this.cfg.showEvent ? this.cfg.showEvent.split(' ').map(e => e + '.tooltip').join(' ') : 'mouseover.tooltip';
```

Comment:
why to switch from enter/leave to over/out?

Reply:
From this MDN tip: https://developer.mozilla.org/en-US/docs/Web/API/Element/mouseenter_event

>With deep hierarchies, the number of mouseenter events sent can be quite huge and cause significant performance problems. In such cases, it is better to listen for mouseover events.

Reply:
They have example showing the difference here: https://developer.mozilla.org/en-US/docs/Web/API/Element/mouseover_event#examples

Reply:
actually now looking at that example mouseenter and leave does seem more correct for this tooltip.  Let me revert.